data/translations/pt_PT.ts

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="pt_PT">
 <context>
+    <name>PictureBox</name>
+    <message>
+        <source>Press to login</source>
+        <translation type="unfinished"></translation>
```

Comment:
I just updated files to current sources, translators can update them without having to use lupdate.